packages/ui/Touchable/Flick.js

```diff
@@ -31,7 +31,7 @@ class Flick {
 		if (!this.tracking) return;
 
 		// This will update the `flickConfig` with the new value
-		this.flickConfig.onFlick = onFlick;
+		this.onFlick = onFlick;
```

Comment:
Could you consider the unit test case to cover this line?

Reply:
I tried to invoke onFlick by simulating flick with touchstart and touchend like onTap.
But it doesn't work. I am not sure flick is simulateable.

Perhaps there was a reasone why Flick.js and Drag.js have low coverage.
I think it is correct to revisit this part again and review it in the direction of increasing the test coverage.